src/pcre2_jit_compile.c

```diff
@@ -13703,14 +13699,17 @@ if (SLJIT_UNLIKELY(opcode == OP_COND) || SLJIT_UNLIKELY(opcode == OP_SCOND))
   if (ccbegin[1 + LINK_SIZE] >= OP_ASSERT && ccbegin[1 + LINK_SIZE] <= OP_ASSERTBACK_NOT)
     {
     SLJIT_ASSERT(has_alternatives);
-    assert = CURRENT_AS(bracket_backtrack)->u.assert;
```

Comment:
Is there a reason to change this? Optimization? Assert is valid here, since the type is an `OP_ASSERT*`?

Reply:
consistency with the other "similar" condition, it also made the intention clearer and obvious that no assert was needed.

will move it back.